framework/Sources/VGSFramework/Core/APIClient.swift

```diff
@@ -11,23 +11,38 @@ import Alamofire
 
 public typealias JsonData = [String: Any]
 public typealias BodyData = [String: Any]
+public typealias HTTPHeaders = [String: String]
 
 class APIClient {
     let baseURL: URL!
     
-    var customHeader: [String: String]?
+    var customHeader: HTTPHeaders?
     
     init(baseURL url: URL) {
         baseURL = url
     }
     
-    func sendRequest(path: String, method: HTTPMethod = .post, value: BodyData, completion block: @escaping (_ data: JsonData?, _ error: Error?) -> Void) {
+    internal static let defaultHttpHeaders: HTTPHeaders = {
         // Add Headers
-        var headers = [
-            "Content-Type": "application/json",
-            "vgs-client": "source=iosSDK&medium=vgs-collect&content=1.0"
+        let version = ProcessInfo.processInfo.operatingSystemVersion
+        let versionString = "\(version.majorVersion).\(version.minorVersion).\(version.patchVersion)"
+        let vgsCollectVersion: String = {
+            guard
+                let vgsInfo = Bundle(for: APIClient.self).infoDictionary,
+                let build = vgsInfo["CFBundleShortVersionString"]
+            else { return "Unknown" }
```

Comment:
Oy wey!
What is the formatting style name? 😜
I believe `swiftlint` not crashing